.travis.yml

```diff
@@ -0,0 +1,45 @@
+language: c
+
+os:
+    - linux
+    - osx
+
+sudo: false
+
+env:
+    # CONDA_JWST_DEPENDENCIES is used because CONDA_DEPENDENCIES is not truly global.
+    global:
+        - MAIN_CMD='python setup.py'
+        - CONDA_CHANNELS='http://ssb.stsci.edu/conda-dev'
+        - CONDA_DEPENDENCIES='pytest nose stsci-jwst'
```

Comment:
Do we need `nose`? There are not many tests right now. Should we just pick `pytest`?